package.json

```
@@ -27,7 +31,9 @@
     "tailwind-merge": "^3.4.0"
   },
   "devDependencies": {
+    "@playwright/test": "^1.59.1",
```

Biome (2.4.10)

[error] The key `@playwright/test` was already declared.

(lint/suspicious/noDuplicateObjectKeys)